kinetic.ignored

```diff
@@ -1,3 +1,2 @@
-chomp_motion_planner
 moveit_experimental
 moveit_planners_chomp
```

Comment:
Why not also release moveit_planners_chomp then?

Reply:
Sorry but I have not caught up with the development status. 
I'll look into the source to see if it's already in releaseable state.

Reply:
I think it's fine to get ` moveit_planners_chomp` released (I posted a bit more detail at https://github.com/ros-planning/moveit/issues/1083#issuecomment-432737000).